bin/build.mjs

```diff
@@ -89,32 +96,53 @@ async function buildPackage(absWorkingDir) {
     const esmContext = await esbuild.context({
       ...buildOptions,
       ...entryPoints('esm'),
-      outdir: 'lib',
       absWorkingDir,
     });
 
-    await esmContext.watch();
+    const extraContext =
```

Comment:
esbuild is clever enough to not bundle mutiple three, so create a bundle-all build

Reply:
> so created a bundle-all build

wdym

Reply:
When using rollup there is a issue that three is bundled multiple times and that issue is lifted after switching to esbuild; therefore It becomes possible to create a build the includes three (not treat three as external to avoid importmap at all).

Reply:
What exactly is "bundle-all" here? Are you simply mentioning the `vrm-viewer.js` build includes Three.js?

Reply:
> Are you simply mentioning the vrm-viewer.js build includes Three.js
yes

Reply:
so now what is the purpose we provide the `separate-three.html` example?

Reply:
This one right? If so I would want to see the actual use cases that combines existing Three.js libraries

https://github.com/pixiv/three-vrm/pull/1395#issuecomment-2128453450

Reply:
I don't think extra example is really needed, this example make most sense by demonstrating it's possible to bring their own three.js or have a build setup.

Reply:
Such an example doesn't make any sense to most users if the purpose is not provided because it looks the same in appearance.

Reply:
To be honest the build is more intended to use with a build setup and we are limited to raw html here. Using Import map itself is a simulation but not a real world usage. So it's more like a test rather than example?

Reply:
Hmm. Yes, I kinda got the intention. Yet still feels like it will cause confusion unless we provide a reason somewhere.

Reply:
We can provide an example that loads an extra model in the scene for example.

```js
import * as THREE from 'three';

const viewer = document.getElementById('viewer');

// See: https://modelviewer.dev/docs/faq.html#entrydocs-general-questions-three
const scene = viewer[Object.getOwnPropertySymbols(viewer).find(e => e.description === 'scene')];

const mesh = new THREE.Mesh(
	new THREE.SphereGeometry(0.1),
	new THREE.MeshStandardMaterial({ color: 0xff8800 }),
);
mesh.position.set(0.5, 0.7, 0);
scene.add(mesh);
```

![image](https://github.com/pixiv/three-vrm/assets/7824814/910d1726-dacc-4fb5-9d38-a9df76f7bcde)

Reply:
Made some changes in https://github.com/pixiv/three-vrm/pull/1395 , what do you think?

Reply:
oh wait, you make `vrm` accessible from `vrmViewer`? That sounds weird a bit considering the original `model-viewer` design doesn't allow Three.js related members to be exposed

Reply:
It's necessary anyway and common when using `lit`. if one want to extend vrm-viewer and call three-vrm specific methods. Doing this via symbols creates more burden than being helpful. Somehow similar to https://github.com/google/model-viewer/blob/master/packages/model-viewer/src/features/scene-graph.ts#L89-L91 ?

Reply:
Yes I agree that changing this to a symbol doesn't make any profit to both end developers and maintainers. However I want to know the intention why `model-viewer` makes such a design decision.